pkg/kubelet/apis/config/validation/util.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/kubelet/stats/pidlimit"
+)
+
+// ParseResourceList parses the given configuration map into an API
+// ResourceList or returns an error.
+func ParseResourceList(m map[string]string) (v1.ResourceList, error) {
```

Comment:
Kubelet config validation doesn't really seem like an appropriate package for this helper, since it relates to a `core/v1` type. `k8s.io/kubernetes/pkg/apis/core/v1/helper/helpers.go` may be more appropriate.